lib/system/index.js

```diff
@@ -24,6 +25,7 @@ var capitalize = function(str) {
 module.exports    = system;
 system.os_name    = os_name;
 system.os_release = os.release();
+system.user_agent = 'Prey/' + version + ' (Node '+ process.version + ' ' + system.os_name[0].toUpperCase() + system.os_name.slice(1) + ' ' + system.os_release + ')';
```

Comment:
My only concern has to do with os_release and os_name warranties. Specifically os_name should provide an array so the slice methods work.